web/src/unread.js

```diff
@@ -667,6 +667,9 @@ export function process_unread_message(message) {
             stream_id: message.stream_id,
             topic: message.topic,
         });
+        if (message.initBy !== undefined) {
+            return;
+        }
     }
```

Comment:
Can you explain why does this check? Also, a code comment for it would be nice.

Reply:
I didn't calculate mention of messages, do I have to?

Reply:
I am not sure @timabbott FYI